doc/devices/ahs_local.rst

```diff
@@ -0,0 +1,101 @@
+The local AHS device
+====================
+
+The local AHS device of the PennyLane-Braket plugin runs Analogue Hamiltonian Simulation on the local Braket SDK. This
+could be either utilizing the processors of your own PC, or those of a `Braket notebook instance <https://docs.aws.amazon.com/braket/latest/developerguide/braket-get-started-create-notebook.html>`_ hosted on AWS.
+
+This device is useful for small-scale simulations in which the time of sending a job to a remote service would add
+an unnecessary overhead. It can also be used for rapid prototyping before running a computation
+on a paid-for remote service.
+
+
+Usage
+~~~~~
+
+After the Braket SDK and the plugin are installed you immediately have access to the local Braket AHS device in PennyLane.
+
+The local AHS device is not gate-based. Instead, it is compatible with `pulse programming <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`_ in PennyLane.
+
+To instantiate the local Braket simulator, simply use:
+
+.. code-block:: python
+
+    import pennylane as qml
+    device_local = qml.device("braket.local.ahs", wires=2)
+
+This device can be used with a QNode within PennyLane. It accepts circuits with a single ``ParametrizedEvolution``
+operator based on a ``ParametrizedHamiltonian`` compatible with the simulated hardware. More information about creating
+PennyLane operators for AHS can be found in the `PennyLane docs <https://docs.pennylane.ai/en/stable/code/qml_pulse.html>`_.
+
+Creating a register
+^^^^^^^^^^^^^^^^^^^
+
+The atom register defines where the atoms will be located, which determines the strength of the interaction
+between the atoms. Here we define coordinates for the atoms to be placed at (in micrometers), and create a constant
+interaction term for the Hamiltonian:
+
+.. code-block:: python
+
+    coordinates = [[0, 0], [0, 5]]
+
+    H_interaction = qml.pulse.rydberg_interaction(coordinates, wires=2)  # wires must match device wires
+
+Creating a drive
+^^^^^^^^^^^^^^^^^^^^^^^
+
+We can create a drive with a global component and (positive) local detunings. If the local detunings are time-dependent,
+they must all have the same time-dependent envelope, but can have different, positive scaling factors.
```

Comment:
After understanding this limitation a bit better I think it'd be good to explain what can and cannot be done for the local terms

Reply:
I think that might be too device-specific for the general documentation, since this isn't supposed to be specific to a particular AHS device. I think it belongs in an example notebook rather than on the plugin overview.